### PR TITLE
DVO-131: Fix default configuration with already watched rules 

### PR DESCRIPTION
**Pull request by ncaak**
### summary
If no custom configuration is found, the project will now whitelist the checks monitored by Rules Team by default.

### fixes
* yaml files containing default configuration data (in case they're used out of the box)
* Updated documentation with default configuration info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,20 @@ DVO performs validation checks using kube-linter. The checks configuration is mi
 
 To configure DVO with a different set of checks, create a ConfigMap in the cluster with the new checks configuration. An example of a configuration ConfigMap can be found [here](./deploy/openshift/configmap.yaml).
 
+If no custom configuration is found (the ConfigMap does not exist or does not contain a check declaration), the project sets the checks to the following list by default:
```
Comment (tremes) [inline README.md:108]: not a native speaker, but what about:
> ..., the operator enables the following checks by default:

Comment (ncaak) [reply README.md]: yeah I went back to re-read it and somehow it's a mess :(

Thanks for the recommendation, already updated it

**Comment by tremes** [comment]
I didn't test this one, but it looks good and it's not so complicated. Thank you.
/lgtm

**Pull request merged**